src/Microsoft.ML.StandardLearners/Standard/MultiClass/Pkpd.cs

```diff
@@ -50,7 +50,7 @@ namespace Microsoft.ML.Trainers
     /// logistic regression is a more principled way to solve a multiclass problem, it
     /// requires that the learner store a lot more intermediate state in the form of
     /// L-BFGS history for all classes *simultaneously*, rather than just one-by-one
-    /// as would be needed for OVA.
+    /// as would be needed for OneVersusAll.
```

Comment:
>OneVersusAll [](start = 31, length = 12)

Please use `see cref=` if you mean a class or just one-versus-all classification model otherwise.